examples/maze/mazelearner.h

```diff
@@ -25,9 +25,7 @@
 #include <cstdint>
 #include <cstdlib>
 #include <ctime>
-#if !(TINYMIND_DISBLE_NEW_LIB)
 #include <new>

```

Comment:
do we really need this include?

Reply:
That gets reverted from the old commits. I'm unsure if you'd prefer to do the "squash and merge" on your side, or if you'd prefer committers to open PRs with squashed branches.

Reply:
Sorry - I bet you are referring to the #include <new>. Yes, we need it. The action_t which is uint8_t is constructed with a new operator in mazelearner. So the library include is necessary here.